kboparai1/ex12/12.py

```diff
@@ -0,0 +1,8 @@
+print "How old are you?",
```

Comment:
Is this document just a primer for ex12? because 11 and 12 are similar but different...